pkg/events/event.go

```diff
@@ -77,6 +77,9 @@ const (
 	MgmtFailed                      = "MgmtFailed"
 	MgmtFailedDiskFull              = "MgmtFailedDiskfull"
 	LowLocalDataAvailSpace          = "LowLocalDataAvailSpace"
+	StartAgentStart                 = "StartAgentStart"
+	StartAgentSucceeded             = "StartAgentSucceeded"
+	StartAgentFailed                = "StartAgentFailed"
```

Comment:
`StartAgentStart` seems like a weird name. Suggested values:
```suggestion
	RunAgentStart                 = "RunAgentStart"
	RunAgentSucceeded             = "RunAgentSucceeded"
	RunAgentFailed                = "RunAgentFailed"
```